Sources/SKSupport/ThreadSafeBox.swift

```diff
@@ -24,7 +24,7 @@ extension NSLock {
 /// A thread safe container that contains a value of type `T`.
 ///
 /// - Note: Unchecked sendable conformance because value is guarded by a lock.
-public class ThreadSafeBox<T>: @unchecked Sendable {
```

Comment:
Out of curiosity, what issues are being caused by it not being `Sendable`?

Reply:
I think the main issue that I noticed was that some of the request handles in `TestSourceKitLSPClient` weren’t sendable. I didn’t technically hit any bug from it but I was wondering why I could modify local variables from the surrounding context from within the request handler, which generally isn’t safe.